doc/source/index.rst

```diff
@@ -187,8 +187,188 @@ the default Docker image to ``rapidsai/rapidsai:latest``, if you're using a cust
+         , idle_seconds_before_scaledown=300
+         , admin_username=admin_username
+         , admin_user_ssh_key=ssh_key_pub
+         , remote_login_port_public_access='Enabled'
+
+         ## UNCOMMENT TO SETUP VIRTUAL NETWORK
+         # , vnet_resourcegroup_name=vnet_rg
+         # , vnet_name=vnet_name
+         # , subnet_name=subnet_name
+      )
+      ct = ComputeTarget.create(ws, ct_name, config)
+      ct.wait_for_completion(show_output=True)
+   else:
+      ct = ws.compute_targets[ct_name]
+
+If your compute target already exists you can call ``ct = ws.compute_targets[ct_name]``.
+
+Setting up vnet
+"""""""""""""""
+If you do not have a virtual network yet there are two ways to create one.
+
+1. Using `https://azure.portal.com <https://azure.portal.com>`_:
+
+   a. On the home page click on `+ Create a resource` on the top-left portion of the page.
+   b. Search for `Virtual Network`.
+   c. Click on `Create` and follow the instructions: select the `Subscription` and the 
+      `Resource group` you will create the vnet in, and provide a `Name` and the `Location`. 
+      **NOTE:** keep the location the same as your AzureML Workspace e.g. if your AzureML Workspace 
+      location is `eastus` you should create the Virtual Network in `East US`.
+   d. Click `Review+Create` and follow through with the rest of the instructions.
+
+2. Using Azure CLI:
+
+   a. Install the AzureCLI; instructions are here: 
+      `https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest 
+      <https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest>`_
+   b. Open terminal and login to your Azure Subscription: ``az login``. This should 
+      automatically log you in into your Azure subscription. **NOTE:** If you have more than 
+      one subscription you will need to set the right subscription to use:
+      ``az account set --subscription "<name-of-sub-to-use>"``
+   c. Create the virtual network: 
+      ``az network vnet create -g <resource-group> -n <vnet-name> --location <location> --subnet-name default``
+
+Define Environment
+~~~~~~~~~~~~~~~~~~
+
+For the `Environment <https://docs.microsoft.com/en-us/python/api/azureml-core/azureml.core.environment.environment?view=azure-ml-py>`_  
+we will use a AzureML curated environment for running Dask CPU cluster, the `AzureML-Dask-CPU`. 
+However, the ``Environment`` class allows you to specify your own docker image and additional packages to install.
+
+.. code-block:: python
+   env = ws.environments[env_name]
+
+Create cluster
+~~~~~~~~~~~~~~
+
+To create cluster:
+
+.. code-block:: python
+
+   amlcluster = AzureMLCluster(
+      workspace=ws,
+      , compute_target=ct
+      , environment_definition=env
+      , initial_node_count=2
+      , admin_username=admin_username
+      , admin_ssh_key=admin_ssh_key_priv   ### path, not contents of the key
+   )
+
+Once the cluster has started, the Dask Cluster widget will print out two links:
```

Comment:
We should also give instructions on how to make this cluster stop. If I restart my notebook kernel or `del` the cluster object what happens, is anything cleaned for me? Do I need to kill the run and delete the compute resource in AzureML myself?